src/batch/src/executor/monitor/stats.rs

```diff
@@ -29,8 +32,17 @@ impl BatchMetrics {
         );
         let row_seq_scan_next_duration = register_histogram_with_registry!(opts, registry).unwrap();
 
+        let exchange_recv_row_number = register_int_counter_vec_with_registry!(
+            "batch_exchange_recv_row_number",
+            "Total number of row that have been received from upstream source",
+            &["up_task_id", "down_task_id"],
```

Comment:
I think it's better to separate the labels to more details: `query_id`, `source_stage_id`, `source_task_id`, `stage_id`, `task_id`

Reply:
The problem with this registry is that we need to clean up the registry after query finished. cc @BowenXiao1999  We need to modify task management to clean up these metrics.

Reply:
Yes, I guess we can do this directly in CN and not invole FE.

Reply:
> I think it's better to separate the labels to more details: query_id, source_stage_id, source_task_id, stage_id, task_id
The "task id" already includes query id and stage id. They are seperated by "-". This is from streaming convention so I think it's good enough now.

Reply:
Think about this case: I want to know all metrics of one query in dashboard, how to create one? Batch query is different from mv since we have may short lived query.

Reply:
I think separate the labels can give a more fine-grained query and it's easy to implement it.